fixtures/MartinGeorgiev/Doctrine/Entity/ContainsIntegers.php

```diff
@@ -6,23 +6,15 @@
 
 use Doctrine\ORM\Mapping as ORM;
 
-/**
- * @ORM\Entity
- */
```

Comment:
I'd rather not pollute the PR with such style refactoring changes, that are not related to the said ORM support functionality. Can you please revert and open a separate PR with these phpDoc changes?

Reply:
Totally happy to do another PR, but this is linked to the changes in `Tests\MartinGeorgiev\Doctrine\ORM\Query\AST\Functions\TestCase` as v3 EM doesn't have an annotation driver. See https://github.com/doctrine/orm/blob/3.0.x/UPGRADE.md

What do you think?